android/res/layout/layout_nav_bottom_numbers.xml

```diff
@@ -3,9 +3,8 @@
   xmlns:android="http://schemas.android.com/apk/res/android"
   xmlns:tools="http://schemas.android.com/tools"
   android:layout_width="match_parent"
-  android:layout_height="@dimen/nav_menu_height"
```

Comment:
So what was the main reason for the bug?

Reply:
Main reason was that part of layout, containing speed, time and distance to final point had fixed height, that was not enough to draw those views correctly. Just set wrap_content in a few places and made some additional cleanup in xml.